app/models/gear.rb

```diff
@@ -1,5 +1,13 @@
 # frozen_string_literal: true
 
+# Gear is equipment that the Hunter can use.
+# Gear includes tags (through the acts_as_taggable_on gem), which are used
+# by the Keeper and Hunter and are visible to users and keywords, which are used
+# to distinguish different sets of gear that might
+# have different numbers of selectable gear, such as with The Flake, who can
+# have one 'normal' weapon and two 'hidden' weapons.  Keywords can also be used
+# to reference modes of transport using the keyword 'transport'.
+
```

Comment:
All user generated top level comments have been moved above the documentation generated by the annotate gem, as it deletes anything between it and the class.